src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -37,6 +38,7 @@
     private static TrackedList<BiomeEntry>[] biomes = setupBiomes();
 
     public static List<Biome> oceanBiomes = new ArrayList<Biome>();
+    private static List<Biome> netherBiomes = Lists.newArrayList(Biomes.field_235254_j_, Biomes.field_235252_ay_, Biomes.field_235253_az_, Biomes.field_235250_aA_, Biomes.field_235251_aB_);
```

Comment:
This will most likely be accessed during parallel mod loading and thus needs to be threadsafe.

Reply:
I'll make it a synchronized list